ramls/order.raml

```diff
@@ -204,3 +204,53 @@ resourceTypes:
     is: [validate]
     post:
       description: Create piece record
+    /{id}:
+      uriParameters:
+        id:
+          description: The UUID of a piece record
+          type: UUID
+      is: [validate, language]
+      put:
+        description: Update a piece record with given {id}
+        body:
+          application/json:
+            type: piece
+            example:
+              strict: false
+              value: !include acq-models/mod-orders-storage/examples/piece_get.sample
+        responses:
+          204:
+            description: "Start value successfully updated"
+          400:
+            description: "Bad request"
+            body:
+              text/plain:
+                example: "Bad request"
+          404:
+              description: "Request id does not exists"
+              body:
+                application/json:
+          500:
+            description: "Internal server error, e.g. due to misconfiguration"
+            body:
```

Comment:
with the earlier comment:
>Also application/json content type will be used if an error handled by mod-orders logic.
I think application/json needs to be added as a return type for 500/400 errors as well